libindi/drivers/telescope/telescope_script.cpp

```diff
@@ -28,7 +28,6 @@
 #include "telescope_script.h"
 
 #define	POLLMS      1000
-#define STATUS_FILE "/tmp/indi-telesope-gateway.status"
```

Comment:
As I understand both locations are OK. But /var/run tends to be used more recently. It is often in memory and non-persistant.